freedom/llh_service/freedom_many_client_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
could this be moved to `scripts/` or `tests/` or similar? As it should not be part of the python package

Reply:
done

Reply:
it appears like there are duplicate files now? I can see that script in both, the new and the old location

Reply:
Oops; intended to remove those, but that must have slipped through the cracks. Should be fixed now